openshift-ci/Dockerfile.registry.build

```diff
@@ -0,0 +1,15 @@
+FROM quay.io/openshift/origin-operator-registry:latest
+FROM registry.access.redhat.com/ubi7-dev-preview/ubi-minimal:latest
+
+
+ARG version=0.1.0
+
+COPY manifests manifests
+COPY deploy/crds/*.yaml manifests/devconsole/${version}/
+
+USER root
+RUN sed -e "s,REPLACE_IMAGE,registry.svc.ci.openshift.org/$(OPENSHIFT_BUILD_NAMESPACE)/stable:devconsole-operator," -i manifests/devconsole/${version}/devconsole-operator.v${version}.clusterserviceversion.yaml
```

Comment:
Since we would be doing a ` FROM root`, the `$(OPENSHIFT_BUILD_NAMESPACE)` would be available..